code/client/include/Client.h

```diff
@@ -0,0 +1,29 @@
+#ifndef COLLATZ_CONJECTURE_CLIENT_H
+#define COLLATZ_CONJECTURE_CLIENT_H
+
+#include <yarp/os/all.h>
+#include <vocabs.hpp>
+#include <ClientIDLInterface.h>
+class Client : public yarp::os::RFModule, ClientIDLInterface{
+private:
+
+    yarp::os::Port rpcPort;
```

Comment:
Is `rpcPort` an `RpcServer` or `RpcClient`?
Specialized classes help developers better catch the use behind.